packages/chatbot-server-mongodb-public/src/systemPrompt.ts

```diff
@@ -69,6 +69,7 @@ const searchContentToolNotes = [
 
 const fetchPageToolNotes = [
   "Fetch the entire page content for a given URL.",
+  `Do not assume that the user wants to use the ${FETCH_PAGE_TOOL_NAME} based on the URL in the Front Matter. The ${FETCH_PAGE_TOOL_NAME} should ONLY be used if the user implies you should look on the page or if the user explicitly provides a URL in their question.`,
```

Comment:
```suggestion
  `Do not assume that the user wants to use the ${FETCH_PAGE_TOOL_NAME} based on URL present in the Front Matter. The ${FETCH_PAGE_TOOL_NAME} should ONLY be used if the user implies you should look on the page or if the user explicitly provides a URL in their question.`,
```

the messages from other clients won't nec have frontmatter, i think. will we need to update prompt elsewhere to take this into account?